src/components/Practice/Mechanics7.jsx

```
@@ -1076,553 +1092,561 @@ const Mechanics7 = ({
+                      <div
+                        onClick={() => {
+                          setIsRecorded(false);
+                          setIsRecording(true);
+                          startRecording(currentText);
+                        }}
+                        style={{
+                          //marginTop: "-10px",
+                          cursor: "pointer",
+                          //marginLeft: "30px",
+                        }}
+                      >
```

ESLint

[error] Visible, non-interactive elements with click handlers must have at least one keyboard listener.

(jsx-a11y/click-events-have-key-events)